backend/typescript/services/implementations/campService.ts

```diff
@@ -551,7 +599,7 @@ class CampService implements ICampService {
       campCounsellors: newCamp.campCounsellors?.map((counsellor) =>
         counsellor.toString(),
       ),
-      campSessions: newCamp.campSessions?.map((session) => session.toString()),
+      campSessions: newCampSessions.map((cs) => cs.id),
```

Comment:
Do we need to cast to string? Is it not an `ObjectID`?

Reply:
newCampSessions is of type `CampSessionDTO[]`, so the camp session's ID is stored as a string there. Previously, we were using the model before but now I am using the result of a helper function